src/components/Changeset/ChangesetDiff.graphql

```diff
@@ -1,7 +1,7 @@
-query ChangesetDiff($changeset: ID!) {
+query ChangesetDiff($changeset: ID!, $resource: ID) {
```

Comment:
Was this added by mistake?

Reply:
yeah something slipped in from other work I was doing